packages/dht/src/connection/websocket/ClientWebsocket.ts

```diff
@@ -27,8 +27,7 @@ export class ClientWebsocket extends EventEmitter<ConnectionEvents> implements I
         this.connectionId = new ConnectionID()
     }
 
-    // TODO explicit default value for "selfSigned" or make it required
-    public connect(address: string, selfSigned?: boolean): void {
+    public connect(address: string, selfSigned: boolean): void {
```

Comment:
Does this setting apply only if we are using secure connection (i.e. `wss`)? 

Does the `false` mean that we don't allow that the certificate is self-signed?  And if `true`, we allow any certificate (self-signed or normal)?
- if this is the case, could rename to `allowSelfSignedCertificate`?